src/components/RangeSlider/components/DualThumb/DualThumb.scss

```diff
@@ -0,0 +1,210 @@
+$stacking-order: (
+  output: 10,
+  input: 20,
+);
+
+$range-track-height: rem(4px);
+$range-thumb-size: rem(24px);
+$range-thumb-border-size: rem(2px);
+$range-thumbs-border: rem(1px) solid color('sky', 'dark');
+$range-thumbs-border-error: rem(2px) solid color('red');
+$range-thumbs-shadow-hover: (0 0 0 rem(1px) rgba(black, 0.4), shadow(faint));
+$range-thumbs-border-active: rem(2px) solid color('indigo');
+
+.Wrapper {
+  position: relative;
+  width: 100%;
+  display: flex;
+}
+
+.TrackWrapper {
+  position: relative;
+  display: flex;
+  align-items: center;
+  width: 100%;
+  min-height: $range-thumb-size;
+
+  &.disabled {
+    opacity: 0.8;
+    cursor: not-allowed;
+  }
+
+  &.error {
+    &:active,
+    &:focus {
+      // stylelint-disable selector-max-specificity
+      // stylelint-disable selector-max-class
+      .Track {
+        --unselected-range: #{color('sky', 'dark')};
```

Comment:
Do we need the actual colors as css vars? 

$unselected-range: color('sky', 'dark');
$selected-range: color('indigo');
$error-selected-range: color('red');

Might make the css simpler?

Reply:
I don't have a strong opinion one way or another. I found it a bit unnecessary to use a variable, as well, but this is the way it was done originally in the single range slider, so it's consistent with that file. It might be worth opening a separate issue to simplify the CSS in both files. I'll skip it here so keep the scope clear.

Reply:
Chatted in Slack and I'm changing it in this PR. 😄

Reply:
I just changed these and it's far less readable. What I do like about keeping them in the variables is that it makes the gradients clearer in terms of which are the selected and unselected areas. What do you think?